slack/utils.bal

```diff
@@ -134,6 +134,22 @@ isolated function getConversationInfo(http:Client slackClient, string channelId,
     }
 }
 
+isolated function getChannelName(http:Client slackClient, string channelId) returns string|error {
+    http:Response response = check slackClient->get(LIST_CONVERSATIONS_PATH);
+    json channelList = check response.getJsonPayload();
+    map<json> channelListJson = <map<json>> channelList;
+    _ = check checkOk(channelListJson);
+    json channels = check channelListJson.channels;
+
+    json[] channelsArr = <json[]> channels;
+    foreach json channel in channelsArr {
+        if channel.id === channelId {
+            return <string> check channel.name;
```

Comment:
Can't we use `ensureType()` here instead of converting to string like this?

Reply:
+1